ocs_ci/utility/kms.py

```diff
@@ -774,11 +774,11 @@ def update_csi_kms_vault_connection_details(update_config):
         csi_kms_conf.get()
     except CommandFailed:
         raise KMSConnectionDetailsError(
-            "CSI kms connection details doesn't exists"
-            "can't continue with update"
+            "CSI kms connection details doesn't exists" "can't continue with update"
```

Comment:
I think that this is missing space between `exists` and `can't` and will lead to `existscan't` . So I think that also when black moved this to one line you can omit the middle `" "`.

Reply:
done